rfcs/20211207-softmax-v2/README.md

```diff
@@ -0,0 +1,144 @@
+# RFC: Softmax v2
+
+## Introduction
+
+Several primitives in the library were designed with operation descriptors containing only `data_desc` member, which covers source and destination memory descriptors.
+There was an assumption that such operations do not change neither memory format layout through execution nor require a different output data type. And would never be.
```

Comment:
"do no change neither" --> "do not change"

Reply:
Jesus Christ... thanks that nobody made my English learning that hard... Applied the suggestion, thanks for educating me.